tests/api.c

```diff
@@ -61370,6 +61370,43 @@ static int test_wolfSSL_X509_STORE_CTX_get0_store(void)
     return EXPECT_RESULT();
 }
 
+#if defined(OPENSSL_ALL) && !defined(NO_RSA) && !defined(NO_CERTS) && \
+    !defined(NO_WOLFSSL_CLIENT) && !defined(NO_BIO) && !defined(NO_TLS)
+static int test_wolfSSL_get_client_ciphers_on_result(WOLFSSL* ssl) {
+    EXPECT_DECLS;
+    if (wolfSSL_is_server(ssl) == 0) {
+        ExpectNull(SSL_get_client_ciphers(ssl));
+    }
+    else {
+        ExpectNotNull(SSL_get_client_ciphers(ssl));
```

Comment:
It would be nice to iterate through the stack and see if we have some sane expected values for the ciphers.

Reply:
Agreed I'll take a look. The testing was light though because it has sanity checks when ran in the updated python 3.12.6 test_ssl.py tests (updated script here https://github.com/wolfSSL/testing/pull/837). Having the cases in our unit test also is useful though, will work on adding some in.